docs/source/faq/visa_distributions.rst

```diff
@@ -0,0 +1,100 @@
+
+VISA external configuration utility for the IVI backend
+========================================================
+
+Managing system VISA resources -
+---------------------------------
+
+The three mentioned VISA distributions all come with a utility to manage VISA
+resource configurations.  This utility is refered to as the external
+configuration utility.  The VISA external configuration utility is used to
+manage instrument attributes used to configure buses, instruments, and aliases.
+This configuration in turn can be used by PyVISA to manage instrument
+configurations in a more user friendly manner than is otherwise afforded.
+
+.. note::
+
+    Not all VISA implementations handle settings configured in the external
+    utility the same.  For instance, in NI-VISA, if a serial port is configured
+    for baudrate = 115000, the default value will still be configured as 9600.
+    To pass the VISA configured value to a new instrument instance, access mode
+    must be set to constants.VI_LOAD_CONFIG (see below), otherwise, the default
+    value of 9600 will be passed to the instrument instance.  R&S-VISA doesn't
+    work this way.  Keysight VISA functionality has not been verified at the
+    time of this writing.
+     
+	open_resource(instr, access_mode=constants.VI_LOAD_CONFIG)
+
+    The important thing to remember is that most values set in the VISA
+    external configuration utility are passed to PyVISA at instantiation of an 
+    instrument by default.  NI-VISA serial ports are an exception that require:
+
+    "open_resource(instr, access_mode=constants.VI_LOAD_CONFIG)".
+
+Set VISA instrument alias -
+---------------------------
+
+A VISA instrument alias is one of the attributes that can be managed by the
+external configuration utility. An alias is another way to refer to an
+instrument.  PyVISA can open an instrument using either the resource name or
+the alias. Resource names are of the form 'GPIB0::3::INSTR', 'ASRL3::INSTR',
+'USB0::0x0699::0x0401::C021095::INSTR'.  What can be observed from these
+examples is that resource names are very hardware dependent.  All three of
+these resource names could refer to a DMM.  If the PyVISA app is coded with the
+resource name then any time the instrument needs to be changed out, say to
+send an instrument for calibration, or the app needs to be propagated to
+multiple systems the names will have to be changed in the app source code.
+Alternately, the alias 'DMM' could be assigned to any of these instruments
+using the external configuration utility.  If the source code uses the alias,
+no source code changes would be required.  One would simply add a new
+compatible instrument with the same alias using the external configuration
+utility.
+
+Aliases provide a more robust way to refer to instruments for applications that
+must be propagated across multiple systems.  It also makes subsituting
+alternate instruments easier when required for maintenance or calibration.
```

Comment:
I am personally a bit bothered by this paragraph. I have developed and deployed several application and I have always preferred to provide this level of abstraction inside the app/in configuration files to avoid the need to rely on an external tool.  So I think I would like to see this toned down a bit.